apps/ledger-live-desktop/src/renderer/drawers/SwapOperationDetails/index.tsx

```diff
@@ -163,7 +165,7 @@ const SwapOperationDetails = ({
             returnTo: "/swap/history",
             accountId: mainAccount.id,
             customDappUrl: undefined,
-            goToURL: "https://buy.moonpay.com/trade_history?ledgerlive",
+            goToURL: `https://buy.moonpay.com/trade_history?ledgerlive&apiKey=pk_live_j5CLt1qxbqGtYhkxUxyk6VQnSd5CBXI&language=${language}&themeId=92be4cb6-a57f-407b-8b1f-bc8055b60c9b`,
```

Comment:
Is this API key fine to be hardcoded? Perhaps an env variable may make more sense?